backend/amd64/cfg_selection.ml

```diff
@@ -431,6 +433,15 @@ let select_operation
     Rewritten (specific (Iprefetch { is_write; addr; locality }), [eloc])
   | _ -> Use_default
 
+let select_operation
```

Comment:
Please make similar changes to arm64. An alternative would be to make these changes in `cfg_selectgen` but there are a few calls and we may need to do something special for builtins, so it currently seems easier to do this check per target.